flink-runtime/src/test/java/org/apache/flink/runtime/taskmanager/TaskManagerRegistrationTest.java

```diff
@@ -533,7 +532,7 @@ protected void run() {
 
 				// kill the first forwarding JobManager
 				watch(fakeJobManager1Gateway.actor());
-				stopActor(fakeJobManager1Gateway.actor());
+				ActorUtils.stopActorsGracefully(fakeJobManager1Gateway.actor());
```

Comment:
can be then just:
```
ActorUtils. stopActor(fakeJobManager1Gateway);
```
also the previous code used `stopActor` not gracefully, which might be intended by the test